BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -29,7 +28,7 @@ class BraintreeClient @VisibleForTesting internal constructor(
      * @suppress
      */
     @get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    val integrationType: String,
+    val integrationType: IntegrationType,
```

Comment:
Can this whole class be internal now?

Reply:
Unfortunately, the feature clients in other modules reference this `integrationType` property, so we wouldn't be able to make it internal without additional refactoring.

Reply:
Ah sorry I meant the whole `BraintreeClient` class - can we restrict it to library scope or no?

Reply:
Oh! BraintreeClient already has the library scope annotation. We can remove them from the individual properties and functions. I'll make an update for that.

Reply:
Updated: 05273983696d48e6c98e1ee507a193543d4c678b